containers/buildAlpine3.docker/Dockerfile

```diff
@@ -8,12 +8,10 @@ COPY ./tools/install-poll.sh /tools/
 RUN /tools/install-poll.sh
 
 COPY ./tools/install-openssl.sh /tools/
-RUN [ "/tools/install-openssl.sh", "1.1.1", "i" ]
-RUN [ "/tools/install-openssl.sh", "1.1.0", "l" ]
+RUN [ "/tools/install-openssl.sh", "1.1.1", "j" ]
 
 COPY ./tools/install-openldap.sh /tools/
-RUN [ "/tools/install-openldap.sh", "1.1.1i" ]
-RUN [ "/tools/install-openldap.sh", "1.1.0l" ]
+RUN [ "/tools/install-openldap.sh", "1.1.1j" ]
```

Comment:
Just curious: why did we install 2 versions before?

Reply:
We had `1.1.0` till 3.4 and just kept theoretical possibility to build even with newer Alpine.